rclcpp/include/rclcpp/subscription_base.hpp

```diff
@@ -227,14 +239,19 @@ class SubscriptionBase : public std::enable_shared_from_this<SubscriptionBase>
   const rosidl_message_type_support_t &
   get_message_type_support_handle() const;
 
-  /// Return if the subscription is serialized
-  /**
-   * \return `true` if the subscription is serialized, `false` otherwise
-   */
+  /// DEPRECATED: Check if subscription takes and handles serialized messages
```

Comment:
We aren't actually deprecating this here, so I wouldn't put this comment in place.  We can do that on Rolling once we merge in the full PR, and there we should use the `[[deprecated]]` annotation.

Reply:
https://github.com/ros2/rclcpp/pull/2165/commits/331b7904ce75c222012e53c038e8a6d0ab3e7242